db/version_builder_test.cc

```diff
@@ -1643,10 +1644,8 @@ TEST_F(VersionBuilderTest, EstimatedActiveKeys) {
   for (uint32_t i = 0; i < kNumFiles; ++i) {
     Add(static_cast<int>(i / kFilesPerLevel), i + 1,
         ToString((i + 100) * 1000).c_str(),
-        ToString((i + 100) * 1000 + 999).c_str(),
-        100U,  0, 100, 100,
-        kEntriesPerFile, kDeletionsPerFile,
-        (i < kTotalSamples));
+        ToString((i + 100) * 1000 + 999).c_str(), 100U, 0, 100, 100,
```

Comment:
This might just be me but this ToString (and the earlier one) look dangerous to me.  Aren't we doing a c_str() on something that is about to be destroyed?

Reply:
This is existing code that got reformatted by clang-format.  I will change if required for acceptance, but noting "not my code".